src/App.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import { Link, Redirect, Route, Switch } from 'react-router-dom';
+
+import { PeopleTable } from './PeopleTable';
+
+import './App.scss';
+
+const App = () => {
+  const [peoples, setPeople] = useState([]);
+
+  const getPeople = () => {
```

Comment:
Instead of passing the `people` array in props, you can fetch people from the server right directly in your `PeopleTable` component